### ISSUE TITLE
FSM transitions seems not to access to the singleotn

### ISSUE THREAD

**Issue by salvob41** [kubecz3k/FiniteStateMachine#15]
After having instantiate a globa singleton http://docs.godotengine.org/en/stable/getting_started/step_by_step/singletons_autoload.html

this seems not be accessed by the transitions node, saying: ` - Compile Error: Identifier not found: global` 
while the States node can access them

**Comment by kubecz3k** [comment]
@salvob41 yes, sorry about that, singletons/autoloads cant be used in tool scripts directly (but they still can be used if you fetch them with `get_node`. 
It's Godot limitation. To workaround please do something like this inside the script:
`var global = get_node("/root/global");`

**Comment by salvob41** [comment]
but it's weird... How can I access it through the States node? Are they not tool scripts?

**Comment by kubecz3k** [comment]
As I said you cant do it directly but you can use get node. (each singleton is an ordinary node that resides as a child of `/root`

**Comment by salvob41** [comment]
Yes, Thank you for your reply.
My comment was trying to make another point though. I'll try to explain better: 
When I try to access the global singleton from a State Node, I can do it. From the transition node, I cannot.
